VERSION

```diff
@@ -1 +1 @@
-PKG_VERSION="2.44.0-bms"
+PKG_VERSION="2.44.0-bms"
```

Comment:
This seems like a stray change…

Reply:
Are you referring to the absence of the trailing newline? ? I'm not sure how that happened but I fixed that.